src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -5113,14 +5113,14 @@ void ExportMusicXml::hairpin(Hairpin const* const hp, staff_idx_t staff, const F
                 tag += "\"diminuendo\"";
             }
             tag += color2xml(hp);
+            tag += positioningAttributes(hp, hp->tick() == tick);
```

Comment:
Is it okay that this was moved into the `if (hp->tick() == tick) {` block?

Reply:
Good question...
I did it because (de-)crescendo lines do it too, but maybe we should rather change those?

Reply:
As far as I can tell it is only needed at the start of a hairpin (and (de-)crescendo line), but yes, it could get simplified to `tag += positioningAttributes(hp, true);` here

Reply:
done that now